### DISCUSSION TITLE
Select all review photos with a shortcut?

### DISCUSSION THREAD

**Discussion by fmunteanu** [photoprism/photoprism#2685] [Q&A] [answered]
I could not find any documentation on Photoprism keyboard shortcuts. Is there a way to select all review photos with a shortcut? Right now I have to click each photo.

**Comment by graciousgrey** [comment, marked as answer]
You can either turn of the review functionality in settings or multi select photos and approve them using the context menu https://docs.photoprism.app/user-guide/navigate/#selection-mode-and-multi-select

**Comment by tdutreui** [comment]
I wanted to do that as well as most of my photos were OK to review, I just rejected a few ones out of thousands.
If you are a techie you could still click the "approve" button programmatically.
This worked for me (use at your own risk, please adapt it to your photoprism version's DOM)
```
document.querySelectorAll('button.action-approve').forEach((e)=>{e.click()})
```
